Cubical/HITs/FiniteMultiset/Recursive.agda

```diff
@@ -0,0 +1,21 @@
+{-# OPTIONS --safe #-}
+module Cubical.HITs.FiniteMultiset.Recursive where
+
+open import Cubical.Foundations.Prelude
+
+open import Cubical.Data.Sigma using (_×_) renaming (_,_ to _,,_)
+open import Cubical.Data.Nat using (ℕ; isSetℕ; _+_; +-comm)
+open import Cubical.HITs.UnorderedPair
+
+private variable
+  ℓ : Level
+  A : Type ℓ
+
+pattern ,_ x = _,,_ _ x
+
++-commPair : UnorderedPair (A × ℕ) → ℕ
++-commPair = rec isSetℕ (λ (, x) (, y) → x + y) λ (, x) (, y) → +-comm x y
```

Comment:
I think this would be easier to read if you use underscore and the usual notation for pairs, i.e. ```_ , x```.

Reply:
I changed to the usual notation.